### DISCUSSION TITLE
Flashpaper

### DISCUSSION THREAD

**Discussion by rnoxy** [selfhst/icons#23] [New Icon Request]
Official website:
https://flashpaper.io/

Github: https://github.com/AndrewPaglusch/FlashPaper

**Comment by shollyethan** [comment]
I'm unable to find a high quality image for FlashPaper. The repo and website both have a small logo that doesn't convert to a square image well when dropping the text. I'll leave this open until I'm able to find something.

**Comment by shollyethan** [comment]
FYI - The FlashPaper dev was super helpful and was able to turn around a high quality icon. It has officially been added to the collection!
